superbench/benchmarks/micro_benchmarks/_export_torch_to_onnx.py

```diff
@@ -0,0 +1,188 @@
+
+        Args:
+            model_name (str): Name of torchvision model to check.
+
+        Returns:
+            bool: True if the model can be exported, False otherwise.
+        """
+        if hasattr(torchvision.models, model_name):
+            return True
+        return False
+
+    def check_benchmark_model(self, model_name):
+        """Check whether can export the benchmark model with given name.
+
+        Args:
+            model_name (str): Name of benchmark model to check.
+
+        Returns:
+            bool: True if the model can be exported, False otherwise.
+        """
+        if model_name in self.benchmark_models:
+            return True
+        return False
+
+    def export_torchvision_model(self, model_name, batch_size=1):
+        """Export the torchvision model with given name.
+
+        Args:
+            model_name (str): Name of torchvision model to export.
+            batch_size (int): Batch size of input. Defaults to 1.
+
+        Returns:
+            str: Exported ONNX model file name.
+        """
+        if not self.check_torchvision_model(model_name):
+            return ''
+        file_name = str(self._onnx_model_path / (model_name + '.onnx'))
+        input_shape = (batch_size, 3, 224, 224)
+        torch.onnx.export(
+            getattr(torchvision.models, model_name)(pretrained=False).eval().cuda(),
+            torch.randn(input_shape, device='cuda'),
+            file_name,
+            opset_version=10,
+            do_constant_folding=True,
+            input_names=['input'],
+            output_names=['output'],
+            dynamic_axes={
+                'input': {
+                    0: 'batch_size',
+                },
+                'output': {
+                    0: 'batch_size',
+                }
+            },
+        )
+        return file_name
+
+    def export_benchmark_model(self, model_name, batch_size=1, seq_length=512):
+        """Export the benchmark model with given name.
+
+        Args:
+            model_name (str): Name of benchmark model to export.
+            batch_size (int): Batch size of input. Defaults to 1.
+            seq_length (int): Sequence length of input. Defaults to 512.
+
+        Returns:
+            str: Exported ONNX model file name.
+        """
+        if not self.check_benchmark_model(model_name):
+            return
+        file_name = str(self._onnx_model_path / (model_name + '.onnx'))
+        input_shape, dtype = (batch_size, seq_length), torch.int64
+        if model_name == 'lstm':
+            input_shape += (self.lstm_input_size, )
+            dtype = None
+        torch.onnx.export(
+            self.benchmark_models[model_name]().eval().cuda(),
+            torch.empty(input_shape, dtype=dtype, device='cuda'),
+            file_name,
+            opset_version=10,
+            do_constant_folding=True,
+            input_names=['input'],
+            output_names=['outputs'],
```

Comment:
`output_names=['outputs']` => ` output_names=['output']`? same with torchvision model

Reply:
for transformers and rnn, there're multiple outputs, maybe just change torchvision model?